se_denseslam/src/bfusion/mapping_impl.hpp

```diff
@@ -258,6 +261,8 @@ struct bfusion_update {
 
     Eigen::Vector3i coord = handler.getNodeCoordinates();
     uint64_t morton_code = compute_morton(coord.x(), coord.y(), coord.z());
+
+    bool isVoxel = std::is_same<DataHandlerT, VoxelBlockHandler<OFusion>>::value;
```

Comment:
Replace OFusion with FieldType to make it generic.

Reply:
changed